hazelcast/src/main/java/com/hazelcast/internal/management/events/ReloadStartedEvent.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2008-2021, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.internal.management.events;
+
+import com.hazelcast.internal.json.JsonObject;
+
+public class ReloadStartedEvent extends AbstractEventBase implements ReloadEvent {
+    @Override
+    public EventMetadata.EventType getType() {
+        return EventMetadata.EventType.RELOAD_STARTED;
+    }
+
+    @Override
+    public JsonObject toJson() {
+        return new JsonObject();
```

Comment:
nit: you can extract it to an `EMPTY_JSON` constant.

Reply:
Since every reload event now has UUID, I didn't need empty json.

https://github.com/hazelcast/hazelcast/pull/20260/commits/210291d1796918dd937a17028bf741418f87667c